### ISSUE TITLE
[Quest][Warlock] Devourer of Souls

### ISSUE THREAD

**Issue by Dudeface23** [dalaranwow/dalaran-wow#4661]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I turned in "Carendin Summons" to Carendin Halgar, but all I can press is complete quest and can not accept the next quest.
**Expected behaviour**: Tell us what should happen instead
He's supposed to offer "Devourer of Souls" but instead does nothing when I talk to him. Without this quest I can't get the succubus summon. This was already discussed in https://github.com/dalaranwow/dalaran-wow/issues/2705 and was closed but it's obviously not working for me so if somebody could tell me what's wrong that'd be great.
**Steps to reproduce the problem**:

1. Start Carendin Summons
2. Go to Carendin Halgar
3. Attempt to complete the quest

**Include proofs for this behaviour**
![wowscrnshot_040817_234331](https://cloud.githubusercontent.com/assets/27120397/24834834/a872d464-1cb5-11e7-9ddf-b2f2db478f89.jpg)
![wowscrnshot_040817_234407](https://cloud.githubusercontent.com/assets/27120397/24834835/abb5bb1e-1cb5-11e7-914b-7deb01907ace.jpg)
![wowscrnshot_040817_235729](https://cloud.githubusercontent.com/assets/27120397/24834905/7b558ae2-1cb7-11e7-9835-01c5e91852bb.jpg)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/
Carendin Summons: 10605
Devourer of Souls: 1472
**Include Screenshots from the issue if necessary**
Already posted above

**Comment by Antiotpad** [comment]
same problem need fix!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
